integration/ct_config.sh

```diff
@@ -24,7 +24,8 @@ for i in $(seq ${num_logs}); do
   # TODO(daviddrysdale): Consider using distinct keys for each log
   tree_id=$(./createtree --admin_server="${ADMIN_SERVER}" --pem_key_path=testdata/log-rpc-server.privkey.pem --pem_key_password=towel)
   echo "Created tree ${tree_id}"
-  sed -i "0,/@TREE_ID@/s/@TREE_ID@/${tree_id}/" "${CT_CFG}"
+  sed -i'.bak' "1,/@TREE_ID@/s/@TREE_ID@/${tree_id}/" "${CT_CFG}"
```

Comment:
Is `-i [suffix]` necessary for it to work under OSX? If yes, would `-i''` do the trick without the backup file? If no, could you please add a comment saying why we need the suffix, lest someone removes it later.